new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,710 @@
+
+    // install operator
+    logger.info("Installing operator in namespace {0}", opNamespace);
+    assertTrue(installOperator(opParams),
+        String.format("Failed to install operator in namespace %s", opNamespace));
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list Helm releases matching operator release name in operator namespace
+    logger.info("Checking operator release {0} status in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+    assertTrue(isHelmReleaseDeployed(OPERATOR_RELEASE_NAME, opNamespace),
+        String.format("Operator release %s is not in deployed status in namespace %s",
+            OPERATOR_RELEASE_NAME, opNamespace));
+    logger.info("Operator release {0} status is deployed in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+
+    // check operator is running
+    logger.info("Check operator pod is running in namespace {0}", opNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for operator to be running in namespace {0} "
+                + "(elapsed time {1}ms, remaining time {2}ms)",
+                opNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(operatorIsReady(opNamespace));
+  }
+
+  /**
+   * Check pod is ready.
+   *
+   * @param podName pod name to check
+   */
+  private void checkPodReady(String podName) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for pod {0} to be ready in namespace {1} "
+                + "(elapsed time {2}ms, remaining time {3}ms)",
+                podName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(podReady(podName, domainUid, domainNamespace));
+  }
+
+  /**
+   * Check service is created.
+   *
+   * @param serviceName service name to check
+   */
+  private void checkServiceCreated(String serviceName) {
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for service {0} to be created in namespace {1} "
+                + "(elapsed time {2}ms, remaining time {3}ms)",
+                serviceName,
+                domainNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(serviceExists(serviceName, null, domainNamespace));
+  }
+
+
+  /**
+   * Login to the WebLogic console and validate its the Home page.
+   *
+   * @param consoleUrl url for the WebLogic console
+   * @param username WebLogic admin username
+   * @param password WebLogic admin password
+   * @throws IOException when connection to console url fails
+   */
+  public static void loginTest(String consoleUrl, String username, String password) throws IOException {
```

Comment:
I thought about this and my PR#1650. You can move this check into the assertion adminNodePortAccessible(int nodePort, String adminUser, String adminPassword), then I can close my PR. Let me know if you think otherwise.

Reply:
moved this login test to adminNodePortAccessible assertion